src/aatoolbox/datasources/fewsnet/fewsnet.py

```diff
@@ -26,23 +26,23 @@
 
 
 def download_fewsnet(
-    date: datetime,
+    date_pub: Union[date, str],
     iso2: str,
     region_name: str,
     region_code: str,
     output_dir: Union[Path, str],
     use_cache=True,
-):
+) -> Path:
     """
     Retrieve the raw fewsnet data.
 
     Depending on the region and date, this data is published per region or
     per country. This function retrieves the country data
-    if it exists, and else the regional data for `date` and `iso2`.
+    if it exists, and else the regional data for `date_pub` and `iso2`.
 
     Parameters
     ----------
-    date : datetime
+    date_pub : date or str
```

Comment:
Sorry, I know this is not in the scope of this PR. But looking a bit more closely at this docstring, it would be helpful to be a bit more specific about some of the parameters, especially `date_pub` (if it's a string, what should the format be?) and `region_name/code` (what are the possible options?)

Reply:
Yes very good point. For the regions I realised due to you that there is only 5 options. So used a `Literal` in the docstring now and added a test to see if the input of `region_name/code` is valid

Reply:
Wow I did not know about using `Literal` that way, I love it!